packages/server/graphql/types/NewMeeting.ts

```diff
@@ -201,6 +201,10 @@ export const newMeetingFields = () => ({
       const viewerId = getUserId(authToken)
       return isMeetingLocked(viewerId, teamId, endedAt, dataLoader)
     }
+  },
+  transcription: {
```

Comment:
-1 I think we could go with this for a prototype, however going forward it's probably much more useful if we keep the context and attach the transcription to the meeting phases/stages

Reply:
Agreed! We could look at the timestamp we get from Recall and add it to the correct phase & stage